De Lijn (Independent Publisher)

Lschockaert opened the pull request:
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

Amjed-Ayoub commented:
Hello @Lschockaert,
Thank you very much and congratulations on your new submission. Our Certification Engineers will review it within 1-2 weeks.
Please let us know if you have any questions.
Thank you very much again for working with us.

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/readme.md:
```diff
@@ -0,0 +1,45 @@
+'This portal is the gateway to the open data sets and webservices of De Lijn. Open Data De Lijn combines freely accessible and usable data on routes, stops, interruptions and timetables.'
+More info on the webpage linked.
+
+## Publisher: Publisher's Name
```
@Lschockaert please repace "Publisher's Name" with your name.

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/apiProperties.json:
```diff
@@ -0,0 +1,24 @@
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
```
@Lschockaert please change this to "#da3b01". please follow this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json)

Lschockaert replied:
I saw this in the pull request sentences but I guess I didn't fully understand the certified vs independent publisher thing.

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "De Lijn Zoek Open Data Services (v1)",
```
@Lschockaert the title exceeds the limit of 30 characters. Please see our requirements [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title)

Lschockaert replied:
Should I add '(Independent Publisher)' to the title? I checked some examples in the github or is 'De Lijn' ok?

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+  "info": {
+    "title": "De Lijn Zoek Open Data Services (v1)",
+    "description": "Get locations of public bus stations and extra info from the Flemish bus company 'De Lijn'",
+    "version": "v1"
```
@Lschockaert please change the version to "1.0".

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+  "paths": {
+    "/zoek/haltes/{zoekArgument}": {
+      "get": {
+        "summary": "zoekHaltes",
```
@Lschockaert all operations summaries should be a valid sentence. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
Please change this for all the operations summaries in the swagger file.

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+          {
+            "in": "path",
+            "name": "zoekArgument",
+            "description": "het zoekargument",
```
@Lschockaert all parameters descriptions should be free of grammatical errors and typos. please see {here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#parameters)
Please change this for all the parameters descriptions in the swagger file.

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+    "/zoek/locaties/{zoekArgument}": {
+      "get": {
+        "summary": "zoekLocaties",
+        "description": "Search locations (x,y) by user adress input. Typical for input on a routeplanner (eg kerkstraat 14 Hasselt or Achter De Kazerne ) . Limitied Geocoding service that retuns stops, adresses or POIs and its XY coordinates",
```
@Lschockaert all files should be free of grammatical errors and typos. Please change the following
"adress" to "address"
"Limitied" to "Limited"
"retuns" to "returns"
"adresses" to "addresses"

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+                "aantalHits": 51,
+                "locaties": [
+                  {
+                    "type": "adres",
```
@Lschockaert can you please explain what you mean by "adres", is it "address"?
if yes may I ask you please to change it in all the swagger file

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/readme.md:
```diff
@@ -0,0 +1,45 @@
+'De Lijn' is the public transport company for busses in Flanders. This is a part of Belgium.
+This service allows you to find a stop/line based on specific terms or location (lat/longitude)
+
+The api that is used is published by 'De Lijn' - I am not the owner of this endpoint and a Power Automate enthousiast.
```
@Lschockaert all files should be free of grammatical errors and typos. Please change the following
"enthousiast" to "enthusiast"

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+{
+  "swagger": "2.0",
+  "info": {
```
@Lschockaert please add a contact section. please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)

Amjed-Ayoub commented on independent-publisher-connectors/De Lijn/De-Lijn-Zoek.swagger.json:
```diff
@@ -0,0 +1,514 @@
+      "API Key": []
+    }
+  ],
+  "tags": []
```
@Lschockaert please add a "x-ms-connector-metadata" section. please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiDefinition.swagger.json)

Lschockaert commented:
@Amjed-Ayoub , I took your feedback and implemented it.
Adres is dutch for address so I changed it where possible as the endpoint is fixed.

Thanks for taking the time to review, learning :)

Amjed-Ayoub commented:
[[certify-connector]]

Amjed-Ayoub commented:
Hello @Lschockaert,
I hope you are doing well.
Congratulations, your pull request is approved. We will move forward with the certification process. We are preparing your connector for production deployment and will queue up for the next deployment schedule. Once your connector onboards the next deployment schedule, it starts to deploy your connector in our production environments that typically takes 3-4 weeks. 
Please let us know if you have any questions.
Thank you very much for working with us.

Amjed-Ayoub commented:
Hello @Lschockaert ,
I hope you are doing well.
Your connector is in production deployment and queued up for the next deployment schedule. This process typically takes 3-4 weeks. Please note that the documentation for your connector will not be generated until the deployment process is in the final steps.
Please let us know if you have any questions.
Thank you very much for working with us.

Amjed-Ayoub commented:
Hello @Lschockaert,
I hope you are doing well.
Congratulations, the deployment of your connector and documentation page was already completed, We have a short [survey](https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR0BHMrjhL0hDmckROosW6AFURERIVUUzMk5XRzJGV1dJM1Q5RVZJUjFXSC4u) on the connector certification process that we would like for you to complete. Your feedback is appreciated and will help us improve the program.
Thank you very much for working with us.
